.eslintrc.json

```diff
@@ -1,18 +1,17 @@
 {
   "globals": {
-    "defineProps": "readonly",
```

Comment:
delete `"defineProps": "readonly",` may cause eslint error in vscode.

error info:  Environment key "vue/setup-compiler-macros" is unknown
see reference:  https://stackoverflow.com/a/69806781

Reply:
I test on a win and a linux environment but still can't reproduce the error. Maybe do a full clean install of eslint?

Reply:
reinstall and no more error notice, maybe related to cache when upgrade packages.